meta-facebook/yv4-sd/src/ipmi/plat_ipmi.c

```diff
@@ -112,3 +119,30 @@ void APP_COLD_RESET(ipmi_msg *msg)
 	msg->data_len = 0;
 	return;
 }
+void OEM_GET_CHASSIS_POSITION(ipmi_msg *msg)
+{
+	CHECK_NULL_ARG(msg);
+
+	msg->completion_code = CC_SUCCESS;
+	msg->data_len = 1;
+
+	uint8_t slot_id = get_slot_id();
+
+	/*   msg->data[0] format:
+	 *
+	 *   Slot Present (Bit 7):
+	 *       0: Slot not present
+	 *       1: Slot present
+	 *
+	 *   Reserved (Bits [6:4])
+	 *
+	 *   Slot ID (Bits [3:0]):
+	 *       0001: Slot 1
+	 *       0010: Slot 2
+	 *       ...
+	 *       1000: Slot 8
+	 */
+
+	msg->data[0] = 0x80 + slot_id;
```

Comment:
Whats 0x80? Can you please create a macro defining the value and then use it?

Reply:
We defined the enum SLOT_STATUS which contains PRESENT(0x80) and NOT_PRESENT(0x00).
Please check. Thanks.

Reply:
Sure, please use these enums on line 146 then, rather than using static 0x80 then.

Reply:
We also change the part.
msg->data[0] = 0x80 + slot_id;
=>
msg->data[0] = SLOT_PRESENT + slot_id;

Please check.
Thanks.